tests.py

```diff
@@ -0,0 +1,77 @@
+import unittest
+import param_calc
+
+class TestParameterCalculator(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.aps = sorted([mains*10 + subs*3 for mains in range(4) for subs in range(10)])
+        
+        cls.ps = [param_calc.calcSkillPoint2Percent(ap) for ap in cls.aps]
+        
+        cls.abilities = [[20.0, 35.0, 80.0],[96.6, 132.3, 138.0],[290.0, 230.0, 170.0],[250.0, 190.0, 130.0],[0.1, 0.55, 1.0],[0.1, 0.55, 1.0],[1.0, 1.0, 1.0],[0.0015, 0.00225, 0.003],[1.05, 0.925, 0.8],[1.05, 0.825, 0.6],[1.0, 0.75, 0.5],[0.4, 0.26, 0.12],[0.2, 0.3, 0.4],[0.72, 0.48, 0.24],[1.3, 1.15, 1.0],[2.4, 2.064, 1.728],[2.4, 2.16, 1.92],[2.4, 2.208, 2.016],[1.0, 0.8, 0.5],[0.6, 0.8, 1.0],[0.65, 0.825, 1.0],[1.0, 1.0, 1.0],[0.5, 0.75, 1.0],[90.0, 180.0, 270.0],[30.0, 60.0, 90.0],[1.44, 1.16, 0.88],[1.44, 1.2, 0.96],[1.44, 1.24, 1.04],[1.3, 1.15, 1.0],[1.25, 1.125, 1.0],[0.65, 0.825, 1.0],[0.7, 0.85, 1.0],[0.8, 0.9, 1.0],[16.8, 14.0, 11.2],[117.0, 148.5, 180.0],[220.0, 410.0, 600.0],[0.5, 0.7, 1.0],[0.55, 0.775, 1.0],[0.6, 0.8, 1.0]]
```

Comment:
shouldnt we try to use the constants from parmeter.json (assuming that this json file does not have any mistakes) for better readability?

Reply:
We could, although there are good arguments against loading files when unit testing.

An alternative for readability is to have a dictionary instead of a list, with descriptive names for the abilities.